app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,10 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
```suggestion
  def home; end
```
check coding convention nào bé ơi

Reply:
resolve mà chưa thấy fix nà